### DISCUSSION TITLE
不支持文字特效吗?导出无报错,导出后无效果

### DISCUSSION THREAD

**Discussion by GerrieWell** [Tencent/libpag#1721] [Q&A] [answered]
类似这种视频里的特效. 文字有生长跳动特效. 导出后文字完全不动了.
导出没有报错.

https://cgmuban.com/ae/ae-template/lyrics-and-voice-over-typography-2/

**Comment by GerrieWell** [comment]
文字图层有 animator 不支持吗?

**Reply by domchen** [reply, marked as answer]
请参考这个列表查看可以直接导出的AE特性列表：https://pag.art/feature.html#ae 所有不在这个列表里的效果都需要使用BMP方式截图导出：https://pag.art/docs/ae-bmp-guide.html
